remote-playback/remote-video-control-pausing-manual.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <title>Test if video is playing on remote device.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/media.js"></script>
+  <style>
+    button {
+        padding: 2em;
+    }
+  </style>
+  <body>
+    <div id="pick-device">
+      <p>
+        Click the button below to prompt for a remote playback device and select
+        one!
+      </p>
+      <p>
+        <button id="prompt-button">Pick device</button>
+      </p>
+    </div>
+    <video src="/media/green-at-15.mp4" id="video"></video>
+    <div id="evaluate" style="display: none">
+      <p>Did the playback on the remote device pause and show the following timestamp? (can vary by some frames)</p>
+      <p id="timestamp" style="font-weight:bold;"></p>
+      <p>
+        <button id="yes">Yes</button>
+      </p>
+      <p>
+        <button id="no">No</button>
+      </p>
+    </div>
+  </body>
+  <script>
+    var v = document.getElementById("video");
+
+    setup({
+      explicit_timeout: true,
+    });
+    async_test((t) => {
+      var button = document.getElementById("prompt-button");
+      button.onclick = t.step_func(() => {
+        promise_test(() => {
+          return v.remote.prompt().then(() => {
+            v.play();
+          });
+        }, "Prompt resolves");
+      });
+
+      var timestampLabel = document.getElementById("timestamp");
+      v.ontimeupdate = function () {
+        var seconds = Math.floor(v.currentTime) + "";
+        var frames = Math.ceil((v.currentTime - seconds) * 30) + "";
+        timestampLabel.innerText = seconds.padStart(2, "0") + ":" + frames.padStart(2, "0");
+        if (v.currentTime >= 2) {
+            v.pause();
+            document.getElementById("evaluate").style.display = "block";
+        }
+      }
+
+      function evaluate(success) {
+        assert_true(success, "Video paused and has correct play position.");
+      }
+      var yesButton = document.getElementById("yes");
+      var noButton = document.getElementById("no");
+      yesButton.onclick = t.step_func_done(function () {
+        evaluate(true);
+      });
+      noButton.onclick = t.step_func_done(function () {
+        evaluate(false);
+      });
+    }, "Test if video is playing on remote device.");
```

Comment:
Please update this with a more specific description, e.g. "Test that pause() on the local video is reflected on the remote device."